test/modules/TestResultCoordinator/TestOperationResultStorage.cs

```diff
@@ -12,12 +12,23 @@ namespace TestResultCoordinator
     using Microsoft.Azure.Devices.Edge.Util;
     using Microsoft.Extensions.Logging;
 
-    class TestOperationResultStorage
+    public class TestOperationResultStorage : ITestOperationResultStorage
     {
         static readonly ILogger Logger = ModuleUtil.CreateLogger("TestResultCoordinator");
-        static Dictionary<string, ISequentialStore<TestOperationResult>> resultStores;
+        private Dictionary<string, ISequentialStore<TestOperationResult>> ResultStores { get; set; }
 
-        public static async Task InitAsync(string storagePath, ISystemEnvironment systemEnvironment, bool optimizeForPerformance, List<string> resultSources)
+        public static async Task<TestOperationResultStorage> CreateStorage(string storagePath, ISystemEnvironment systemEnvironment, bool optimizeForPerformance, List<string> resultSources)
```

Comment:
We should pass in the DbStoreProvider... that way we can use RocksDb / InMemoryStore depending on config.

Reply:
Why would that be something we want configured in our tests? I can't think of why we wouldn't want to use persistent storage?